dspace/config/spring/api/core-services.xml

```diff
@@ -179,5 +179,7 @@
     <bean class="org.dspace.app.statistics.clarin.ClarinMatomoBitstreamTracker"/>
     <bean class="org.dspace.app.statistics.clarin.ClarinMatomoOAITracker"/>
 
+    <bean class="org.dspace.core.ProvenanceServiceImpl"/>
```

Comment:
why?

Reply:
If this PR were to apply to Vanilla, we should use a Spring service and access it using @Autowired instead of creating 4-5 new instances in the app.